src/quick-start.md

```diff
@@ -0,0 +1,295 @@
+kind: AdmissionPolicy
+metadata:
+  name: privileged-pods
+spec:
+  module: registry://ghcr.io/chimera-kube/policies/pod-privileged:v0.1.1
+  resources:
+  - pods
+  operations:
+  - CREATE
+  - UPDATE
+  settings:
+    trusted_users:
+    - alice
+```
+
+This is a quick overview of the attributes of the `AdmissionPolicy` resource:
+
+* `module`: this is the location of the Chimera policy, several schemas are
+  supported.
+  * `registry`: download from an [OCI artifacts](https://github.com/opencontainers/artifacts)
+    compliant container registry
+  * `http`, `https`: download from a regular HTTP(s) server
+  * `file`: load the module from the local filesystem
+* `resources`: types of resources evaluated by the policy
+* `operations`: what operations for the previously given types should
+  be forwarded to this admission policy by the API server for
+  evaluation.
+* `settings` (optional): a free-form object that contains the policy
+  configuration values.
+* `failurePolicy` (optional): how unrecognized errors and timeout errors from
+  the policy are handled. Allowed values are `Ignore` or `Fail`. `Ignore` means
+  that an error calling the webhook is ignored and the API request is allowed
+  to continue. `Fail` means that an error calling the webhook causes the
+  admission to fail and the API request to be rejected.
+  The default behaviour is `Fail`.
+
+The complete documentation of this Custom Resource can be found
+[here](https://github.com/chimera-kube/chimera-controller/blob/main/docs/crds/README.asciidoc)
+or on
+[docs.crds.dev](https://doc.crds.dev/github.com/chimera-kube/chimera-controller).
+
+> **NOTE:** Admission policies are registered with a `*` webhook
+> `scope`, which means that registered webhooks will be forwarded all
+> requests matching the given `resources` and `operations` -- either
+> namespaced (in any namespace), or cluster-wide resources.
+
+> **NOTE:** The `AdmissionPolicy` resource is cluster-wide. There are
+> plans to also provide a namespaced version that will only impact
+> registered namespaced resources on its own namespace.
+
+## Enforce your first policy
+
+We will use the [`pod-privileged` policy](https://github.com/chimera-kube/pod-privileged-policy).
+This policy regulates who can create privileged containers inside of a Kubernetes cluster.
+Only a chosen set of users, or groups of users, will be granted the ability to
+create privileged containers.
+
+Let's define a `AdmissionPolicy` that allows only the members of the
+`the_trusted_ones` group to create privileged containers:
+
+```console
+kubectl apply -f - <<EOF
+apiVersion: chimera.suse.com/v1alpha1
+kind: AdmissionPolicy
+metadata:
+  name: privileged-pods
+spec:
+  module: registry://ghcr.io/chimera-kube/policies/pod-privileged:v0.1.1
+  resources:
+  - pods
+  operations:
+  - CREATE
+  - UPDATE
+  settings:
+    trusted_groups:
+    - the_trusted_ones
+EOF
+```
+
+This will produce the following output:
```

Comment:
But this doesn't show the actual status. How do I verify that the policy-server is actually running ?

Reply:
Written some lines below:

> Once all the instances of `policy-server` are ready, the
`ValidatingWebhookConfiguration` will be visible:
> 
> ```console
> $ kubectl get validatingwebhookconfigurations.admissionregistration.k8s.io
> NAME              WEBHOOKS   AGE
> privileged-pods   1          9s
>```